testdata/networking/ptp_master.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: ptpconfig-master
+  namespace: openshift-ptp
+spec:
+  profile:
+  - name: "ptp-master-profile"
+    interface: "eno2"
+    ptp4lOpts: "-2"
+    phc2sysOpts: "-a -r"
+  recommend:
+  - profile: "ptp-master-profile"
+    priority: 99
+    match:
+    - nodeName: "dell-per740-14.rhts.eng.pek2.redhat.com"
```

Comment:
this look like internal machine...is it always going to be on?

Reply:
This is just a default value, it will always be replaced when using the file.

Reply:
We shouldn't expose internal machine names.  We should use something from `example.com` as it is reserved for such purposes.

Reply:
Thanks @rbbratta , fixed it.